tests/s3/versioning/test_get_put_object_tagging.py

```diff
@@ -290,3 +290,406 @@ def test_get_put_obj_tags_40429(self):
+                                             bucket_name=self.bucket_name,
+                                             object_name=self.object_name,
+                                             version_tag=self.ver_tag, versions_dict=self.versions,
+                                             version_id=latest_ver_id1)
+        assert_utils.assert_true(resp[0], resp)
+
+        LOGGER.info("Step 10: Perform GET Object Tagging for %s with versionId=%s",
+                    self.object_name, latest_ver_id1)
+        put_tag = self.ver_tag[self.object_name][latest_ver_id1][-1]
+        resp = s3_cmn_lib.get_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                             s3_ver_test_obj=self.s3_ver_obj,
+                                             bucket_name=self.bucket_name,
+                                             object_name=self.object_name,
+                                             version_id=latest_ver_id1)
+        assert_utils.assert_true(resp[0], resp)
+        get_tag = resp[1]['TagSet'][0]
+        assert_utils.assert_equal(get_tag, put_tag, "Mismatch in tag Key-Value pair."
+                                                    f"Expected: {put_tag} \n Actual: {get_tag}")
+
+        LOGGER.info("Step 11: Perform GET Object Tagging for %s with versionId=%s",
+                    self.object_name, latest_ver_id2)
+        put_tag = self.ver_tag[self.object_name][latest_ver_id2][-1]
+        resp = s3_cmn_lib.get_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                             s3_ver_test_obj=self.s3_ver_obj,
+                                             bucket_name=self.bucket_name,
+                                             object_name=self.object_name,
+                                             version_id=latest_ver_id2)
+        assert_utils.assert_true(resp[0], resp)
+        get_tag = resp[1]['TagSet'][0]
+        assert_utils.assert_equal(get_tag, put_tag, "Mismatch in tag Key-Value pair."
+                                                    f"Expected: {put_tag} \n Actual: {get_tag}")
+
+        LOGGER.info("Step 12: Perform GET Object Tagging for %s without versionId specified",
+                    self.object_name)
+        resp = s3_cmn_lib.get_object_tagging(s3_tag_test_obj=self.s3_tag_obj,
+                                             s3_ver_test_obj=self.s3_ver_obj,
+                                             bucket_name=self.bucket_name,
+                                             object_name=self.object_name)
+        assert_utils.assert_true(resp[0], resp)
+        get_tag = resp[1]['TagSet'][0]
+        assert_utils.assert_equal(get_tag, put_tag, "Mismatch in tag Key-Value pair."
+                                                    f"Expected: {put_tag} \n Actual: {get_tag}")
+
+        LOGGER.info("Step 13: Perform PUT Object for %s", self.object_name)
+        resp = self.s3_test_obj.put_object(bucket_name=self.bucket_name,
```

Comment:
You would need to call upload_version with chk_null_version=True here, since bucket is Suspended